.github/labeler.yml

```
@@ -191,11 +191,10 @@ utils.masked:
     - docs/utils/masked/**/*
 
 visualization:
-- any:
-  - changed-files:
-    - any-glob-to-any-file:
-      - '**/visualization/**/*'
-      - '!**/visualization/wcsaxes/**/*'
+- changed-files:
+  - any-glob-to-any-file:
+    - '**/visualization/**/*'
+    - '!**/visualization/wcsaxes/**/*'
```

Okay here's the actual fix:
```suggestion
- any:
   - changed-files:
     - all-globs-to-any-file:
       - '**/visualization/**/*'
       - '!**/visualization/wcsaxes/**/*'
```

For reference, here are my test PRs on my fork
- https://github.com/neutrinoceros/astropy/pull/11
- https://github.com/neutrinoceros/astropy/pull/12